projboard/tests/test_client.py

```diff
@@ -14,6 +14,8 @@ def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
+    # CHECK IF THE ARTICLES CONTAINTS THE CORRECT NUMBER OF ARTICLES
+    assert len(response.context['articles']) == Article.objects.count()
```

Comment:
I think we should avoid to check the db because it's a client test.
Take a look [here](https://github.com/redhat-beyond/Smarticle/pull/70#discussion_r1058195094)

Reply:
@rawadabu you can make a fixture and just call it instead of making queries directly, although sometimes we can make direct queries in testing .

Reply:
@Anas-mhameed @Shalevro2 So you suggest like to add a function to `Article` that gets me all the articles, and to use it instead of direct queries?

Reply:
I suggest you make a fixture in confest.py file and just use it 
There is no need to make such function Article model

Reply:
Great .
@Shalevro2 ???

Reply:
Yeah, I think it's better.